migration/1721730420883-AddressLabel.js

```diff
@@ -0,0 +1,15 @@
+const { MigrationInterface, QueryRunner } = require("typeorm");
+
+module.exports = class AddressLabel1721730420883 {
+    name = 'AddressLabel1721730420883'
+
+    async up(queryRunner) {
+        await queryRunner.query(`ALTER TABLE "wallet" ADD "displayName" nvarchar(256)`);
+        await queryRunner.query(`ALTER TABLE "user" ADD "label" nvarchar(256)`);
+    }
+
+    async down(queryRunner) {
+        await queryRunner.query(`ALTER TABLE "user" DROP COLUMN "label"`);
+        await queryRunner.query(`ALTER TABLE "wallet" DROP COLUMN "displayName"`);
```

Comment:
Add `"dbo"`

For example: 
`ALTER TABLE "dbo"."wallet" ADD "displayName" nvarchar(256)`
`ALTER TABLE "dbo"."user" ADD "label" nvarchar(256)`

Same for the down part

Reply:
Should work without `dbo` IMO